modules/parrableIdSystem.js

```diff
@@ -183,6 +184,8 @@ function fetchId(configParams) {
   const eid = (parrableId) ? parrableId.eid : null;
   const refererInfo = getRefererInfo();
   const uspString = uspDataHandler.getConsentData();
+  const gdprApplies = (gdprConsentData && typeof gdprConsentData.gdprApplies === 'boolean' && gdprConsentData.gdprApplies);
+  const gdprConsentString = (gdprConsentData && gdprApplies && gdprConsentData.consentString) || '';
```

Comment:
Just a tip for future reference, you could have done this in a shorter way.  
You could have changed:

```
typeof gdprConsentData.gdprApplies === 'boolean' && gdprConsentData.gdprApplies
```

into:

```
gdprConsentData.gdprApplies === true
```

Also, for line 188, you don't need to check for `gdprConsentData`, if that evaluates to `false` then `gdprApplies` would be also `false`. So, for line 188 it's enough just to do:

```
const gdprConsentString = (gdprApplies && gdprConsentData.consentString) || '';
```